### PR TITLE
fix README.md clearly

### PR DESCRIPTION
**Pull request by waiterQ**
update jobflow README congruously.

### CODE REVIEW

File: README.md
Comment (william-wang) [inline README.md:59]: Do not need this line here. please lease it as it is.

**Comment by waiterQ** [comment]
> Why make these changes? Have some instructions?

from [website/issue/268](https://github.com/volcano-sh/website/issues/268), i think README should more congruously, jobflow example need more friendly to users

File: README.md
Comment (waiterQ) [reply README.md]: done

File: docs/design/jobflow/README.md
Comment (waiterQ) [reply docs/design/jobflow/README.md]: done

File: example/jobflow/README.md
Comment (hwdef) [inline example/jobflow/README.md:20]: jobflow has been donated to the volcano community to become part of the volcano project. The relevant PR is under review.
https://github.com/volcano-sh/volcano/pull/2383
I don't think it's the right time to change this document at the moment, and wait until the jobflow code is fully integrated before optimizing the documentation.

/hold

Comment (waiterQ) [reply example/jobflow/README.md:20]: so..., no need for example/jobflow/README.md? what about README.md, if no-need change, i will close this pr

Comment (hwdef) [reply example/jobflow/README.md:20]: This pr can stay open.I think it needs some modification, after that we can merge this pr, and after jobflow is merged, it may need to be modified again.

Comment (william-wang) [reply example/jobflow/README.md:20]: @hwdef Good suggestion:). We can provide this example doc for our users and update it right after #2383 get merged.

File: README.md
Comment (hwdef) [inline README.md:33]: It is not recommended to delete all introductions related to jobflow in the readme. You can keep the brief introduction and direct it to the design doc you write.

File: example/jobflow/README.md
Comment (waiterQ) [reply example/jobflow/README.md:20]: ok.

File: README.md
Comment (waiterQ) [reply README.md:33]: got it, after #2383, i will modify this pr to get to suitable

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**
